crypto/store/store_lib.c

```diff
@@ -935,6 +935,11 @@ OSSL_STORE_CTX *OSSL_STORE_attach(BIO *bp, const char *scheme,
     if ((loader = ossl_store_get0_loader_int(scheme)) != NULL)
         loader_ctx = loader->attach(loader, bp, libctx, propq,
                                     ui_method, ui_data);
+    // ossl_store_get0_loader_int will will raise an error if the loader for
```

Comment:
Please do not use C++ style comments and please properly indent after if.

Reply:
Sorry about that, I'll update shortly. Thanks